main_view.html

```diff
@@ -11,37 +11,39 @@
 
     
 <header>
-<div class="taskbar"> <span class="time"> 9:41 </span>  
-<img  class="status_phone" src="images/Cellular_Connection.svg" alt="Connection">  
-<img   class="status_phone" src="images/Wifiwifi_signal.svg" alt="Wifi_signal">
-<img class="status_phone" src="images/Batterybattery.svg" alt="Battery">
-</div>
-<div class="menu"> <span>Welcome</span> <img src="images/Enabledmenu_bar.svg" alt=""></div>
+<div class="menu">
+  <a href="#" class="logo"> Welcome </a>  <nav class="menu_img"><a href="#"><img src="images/Enabledmenu_bar.svg" alt=""> </a></nav>
+    </div>
 </header>
```

Comment:
- [ ] Kindly put every tag in a separate line for a cleaner code, which make it easier for other to read it.
- [ ] Good job making the logo a link but please remove the `a` tag styling, right now it shows an underlined logo.
- [ ] Please know that the menu_bar shouldn't be inside a link `a` tag as it isn't a link, kindly fix that.